proteinworkshop/constants.py

```diff
@@ -47,3 +47,52 @@
 DEFAULT_LOG_FILE = PROJECT_PATH / "logs" / "default_log.log"
 DEFAULT_LOG_FILE.parent.mkdir(parents=True, exist_ok=True)
 DEFAULT_LOG_LEVEL = logging.DEBUG  # verbose logging per default
+
+# ---------------- VALID DATASETS CONSTANTS ---------
+FOLDCOMP_DATASET_NAMES = [
+    "afdb_rep_v4",
+    "afdb_rep_dark_v4",
+    "afdb_swissprot",
+    "afdb_swissprot_v4",
+    "afdb_uniprot_v4",
+    "esmatlas",
+    "highquality_clust30",
+    "a_thaliana",
+    "c_albicans",
+    "c_elegans",
+    "d_discoideum",
+    "d_melanogaster",
+    "d_rerio",
+    "e_coli",
+    "g_max",
+    "h_sapiens",
+    "m_jannaschii",
+    "m_musculus",
+    "o_sativa",
+    "r_norvegicus",
+    "s_cerevisiae",
+    "s_pombe",
+    "z_mays",
+]
```

Comment:
We should probably add a docstring for these & add constants to the docs.

Reply:
Great idea, on it 👍